tools/cosmovisor/README.md

```diff
@@ -339,17 +333,23 @@ Initialize cosmovisor with the current binary:
 cosmovisor init ./build/simd
 ```
 
-Now you can run cosmovisor with simapp v0.44:
+Now you can run cosmovisor with simapp v0.47.4:
 
 ```shell
 cosmovisor run start
 ```
 
-#### Update App
+### Update App
+
+Update app to the latest version (e.g. v0.50.0).
+
+:::note
 
-Update app to the latest version (e.g. v0.45).
+Migration plans are defined using the `x/upgrade` module and described in [In-Place Store Migrations](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/core/15-upgrade.md). Migrations can perform any deterministic state change.
 
-Next, we can add a migration - which is defined using `x/upgrade` [upgrade plan](https://github.com/cosmos/cosmos-sdk/blob/main/docs/core/upgrade.md) (you may refer to a past version if you are using an older Cosmos SDK release). In a migration we can do any deterministic state change.
+The migration plan to upgrade the simapp from v0.47 to v0.50 is defined in `simapp/upgrade.go`.
```

Comment:
Can we maybe still link the x/upgrade module docs?

Reply:
I omitted the `x/upgrade` module doc link because it is already linked on the `In-Place Store Migrations` page. 

I can add it back if you prefer. 

WDYT?

Reply:
Alright makes sense 👍